ipa-server/README.md

```diff
@@ -0,0 +1,25 @@
+## Prerequisites
+
+This app has the following prereq:
+
+* OpenShift Command Line Tool
+* [Openshift Applier](https://github.com/redhat-cop/openshift-applier/). As a result you'll need to have [ansible installed](http://docs.ansible.com/ansible/latest/intro_installation.html)
+* The permissions to grant an SA the anyuid scc.
+
+### Environment Setup
+
+1. Clone this repository: `git clone https://github.com/redhat-cop/containers-quickstarts`
+2. `cd containers-quickstarts/ipa-server`
+3. Run `ansible-galaxy install -r requirements.yml --roles-path=roles`
+4. Login to OpenShift: `oc login -u <username> https://console.example.com:8443`
+
+:heavy_exclamation_mark: freeipa-server container will run under a replica of `anyuid` SCC. You should ensure you are logged into the Cluster (step 4) with an user with privileges to create SecurityContextConstraints with the required scope
```

Comment:
This is ok for now, but I'd like to see a follow-up PR where we better define a custom role/rolebinding that allows for this to run without needing to run under `anyuid`

Reply:
Agreed. This is just in place for the initial commit. I think this is possible, but will require some untangling of their init script (and likely some buildconfig setup)

Reply:
Perfect